project.clj

```diff
@@ -31,10 +31,10 @@
                  [net.sf.opencsv/opencsv "2.3"]
                  [de.ubercode.clostache/clostache "1.4.0" :exclusions [org.clojure/core.incubator]]
                  [slingshot "0.12.2"]
-                 [org.cyverse/clj-icat-direct "2.8.2"
+                 [org.cyverse/clj-icat-direct "2.8.4-SNAPSHOT"
                    :exclusions [[org.slf4j/slf4j-log4j12]
                                 [log4j]]]
-                 [org.cyverse/clj-jargon "2.8.5"
+                 [org.cyverse/clj-jargon "2.8.7-SNAPSHOT"
```

Comment:
hopefully I'm not just being a broken record/you may already be on this wavelength, so bear with me, but: sooner than later for getting rid of any SNAPSHOT versions in project.clj -- versions are cheap, may as well have more released versions of our libraries if we feel like it!

Reply:
Yep, I was planning to switch to non-snapshot versions after those pull requests were approved.